docs/layout-of-source-files.rst

```diff
@@ -28,6 +28,9 @@ it does include the supplied string in the :ref:`bytecode metadata <metadata>`.
 If you do not want to specify a license or if the source code is
 not open-source, please use the special value ``UNLICENSED``.
 
+.. note::
```

Comment:
I would not make this stand out as a note, because then it becomes more prominent than the paragraph above. What about just saying that ``UNLICENSED`` is different than "the UNLICENSE" and that we follow the recommendation by npm (and link `https://docs.npmjs.com/cli/v7/configuring-npm/package-json#license` instead of the github issue)